styles.less

```diff
@@ -30,3 +30,7 @@ atom-text-editor {
 atom-text-editor::shadow .cursor {
   // border-color: red;
 }
+
+.tab .title:before {
```

Comment:
Does this hide just the new icons or also the find results icon? For very different window types (where the icon itself will tell you what the window is) I kinda like it...